src/platforms/javascript/guides/ember/index.mdx

```diff
@@ -6,60 +6,115 @@ redirect_from:
   - /clients/javascript/integrations/ember/
 ---
 
-To use Sentry with your Ember application, you will need to use Sentry’s browser JavaScript SDK: `@sentry/browser`.
+On this page, we provide concise information to get you up and running with Sentry's Ember SDK, automatically reporting errors and exceptions in your application.
 
-On its own, `@sentry/browser` will report any uncaught exceptions triggered from your application.
-In order to use ESM imports without any additional configuration, you can use `ember-auto-import`
-by installing it with `ember install ember-auto-import`.
+**Note:** If you don't already have an account and Sentry project established, head over to [sentry.io](https://sentry.io/signup/), then return to this page.
 
-Starting with version `5.x` our `Ember` integration lives in it's own package `@sentry/integrations`.
-You can install it with `npm` / `yarn` like:
+Our Ember SDK currently supports all lts versions of Ember 3.x.
 
-```bash {tabTitle:npm}
-npm install --save @sentry/browser @sentry/integrations
+## Install
```

Comment:
Note this WIP - if you merge changes prior to my finishing, I will adapt your content - https://github.com/getsentry/sentry-docs/pull/2274